e2e_test/test/e2e_test.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+void main() {
+  group('setup test', () {
```

Comment:
I guess I didn't real close enough the first time - we do already have dazel init in a setUp. I guess it's not *that* slow, but I think it would be nice to move it to setUpAll rather than repeat it.